src/components/theme/ContentMetadataTags/ContentMetadataTags.jsx

```diff
@@ -45,10 +45,16 @@ const ContentMetadataTags = (props) => {
 
   const contentImageInfo = getContentImageInfo();
 
+  let title_tag_content = seo_title || title;
+  let nav_root_title = props.content['@components']?.navroot?.navroot?.title;
```

Comment:
Hi @erral, I see that you hardcoded the introduction of the site title/language root title here. I was thinking that maybe it is better to work as this https://github.com/plone/volto/issues/4723, as it is more customizable. I already did this in my pr, which is now closed, could you include this? Check the getTitle() function from https://github.com/plone/volto/pull/4862/files

Reply:
With this approach, you allow to change the separator character though settings. This might be better because other projects that use volto can avoid the customization of this.

Reply:
I like the approach @dobri1408

Reply:
@dobri1408 I have reworked this with your suggestion. 

now the default setting is not to show the site title (and thus it's not a breaking change).

Moreover the site title is not shown if it is equal to the page title (like it is done in Classic)